content/docs/handling-events.md

```diff
@@ -1,72 +1,73 @@
 ---
 id: handling-events
-title: Handling Events
+title: Իրադարձությունների մշակում
 permalink: docs/handling-events.html
 prev: state-and-lifecycle.html
 next: conditional-rendering.html
 redirect_from:
   - "docs/events-ko-KR.html"
 ---
+Իրադարձությունների մշակումը React-ի էլեմենտներում շատ նման է իրադարձությունների մշակմանը DOM էլեմենտներում։ Կան որոշ շարահյուսական տարբերություններ։
 
-Handling events with React elements is very similar to handling events on DOM elements. There are some syntactic differences:
+* React-ում իրադարձությունների անունները փոքրատառի փոխարեն ուղտաԳիր են։
+* JSX-ում  տողային ֆունկցիայի փոխարեն, որպես իրադարձություն մշակող դուք փոխանցում եք ֆունկցիա։
 
-* React events are named using camelCase, rather than lowercase.
-* With JSX you pass a function as the event handler, rather than a string.
-
-For example, the HTML:
+Օրինակ, HTML-ը՝
 
 ```html
 <button onclick="activateLasers()">
-  Activate Lasers
+  Ակտիվացնել լազերնրենը
 </button>
 ```
 
-is slightly different in React:
+React-ում մի փոքր այլ է՝
 
 ```js{1}
 <button onClick={activateLasers}>
-  Activate Lasers
+    Ակտիվացնել լազերնրենը
 </button>
 ```
 
-Another difference is that you cannot return `false` to prevent default behavior in React. You must call `preventDefault` explicitly. For example, with plain HTML, to prevent the default link behavior of opening a new page, you can write:
+Մեկ այլ տարբերությունն է այն, որ React-ում դուք չեք կարող վերադարձնել `false`, որպեսզի կանխեք իսկզբանե նախատեսված գործողությունը։
+Ակնհայտորեն անհրաժեշտ է կանչել `preventDefault`։ Օրինակ՝ պարզ HTML-ում, դիտարկիչում հղումը նոր էջում բացելու գործողությունը կանխելու համար դուք կարող եք գրել՝
```

Comment:
Եթե ընթերցողը կարդալու ընթացում Ակնհայտորեն բառից հետո դադար առնի (ստորակետ դնի), լրիվ իմասը կփոխվի։ Այսինքն ընթերցողը պիտի հասկանա, որ ոչ թե ակնհայտ ա, որ անհրաժեշտ ա կանչել ինչ-որ բան, այլ էտ ինչ-որ բանին պիտի կանչենք ակնհայտորեն։ Ու առհասարակ բացահայտ-ը միգուցե ավելի լավ թարգմանություն ա։

Այսինքն, You must call preventDefault explicitly. <- սրա թարգմանությունը.

Անհրաժեշտ է կանչել preventDefault-ը բացահայտորեն։ (Դուք պետք է կանչեք preventDefault-ը բացահայտորեն։)
կամ
Անհրաժեշտ է բացահայտ կանչել preventDefault-ը։ (Դուք պետք է բացահայտ կանչեք preventDefault-ը։)

Reply:
done